src/entities/monitor-method.entity.ts

```diff
@@ -0,0 +1,47 @@
+import { BaseEntity, Entity, Column, PrimaryColumn, Unique } from 'node_modules';
```

Comment:
this should be...
import { BaseEntity, Entity, Column, PrimaryColumn, Unique } from 'typeorm';